### ISSUE TITLE
Unknown class error when using with proguard

### ISSUE THREAD

**Issue by silentninja** [TheJuki/KFormMaster#80]
## Description
Getting the following error when using it with proguard.
Note: the configuration refers to the unknown class 'com.thejuki.kformmaster.FormActivityTest'

### KFormMaster version

* Version: 5.0.7

**Comment by TheJuki** [comment]
Are you using KFormMaster from Bintray? I just tested Version: 5.0.7 with a release build with proguard and did not have this issue.

**Comment by silentninja** [comment]
Yes, I am using version 5.0.7 from Jcenter.  Build tools version is com.android.tools.build:gradle:3.3.0-rc01. I suppressed it using -dontnote, haven't got any error when using the app though.

**Comment by TheJuki** [comment]
I'l return to this once the Android Gradle plugin has been updated. https://developer.android.com/studio/releases/gradle-plugin
